ImgShareData/ImgurAuthorizationModels/IImgurAuthorization.cs

```diff
@@ -0,0 +1,46 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace ImgShare.APISource.Data.ImgurAuthorization
+{
+    public interface IImgurAuthorization

```

Comment:
nit: what is this/why is it being flagged as addition in diff [dab5fc0613e031a](https://github.com/gfmatthews/imgurfordotnet-windows/pull/7/commits/dab5fc0613e031a96eadffe1a606d925408ba0de)